devicon.json

```
@@ -345,6 +345,24 @@
         "color": "#c4473a",
         "aliases": []
     },
+    {
+        "name": "angularmaterial",
+        "altnames": [],
+        "tags": [
+            "framework",
+            "javascript"
+        ],
+        "versions": {
+            "svg": [
+                "original"
```

Plain can also be added here :)
```suggestion
                "original",
                "plain"
```